src/V3PerPackage/App.config

```diff
@@ -1,6 +1,3 @@
 <?xml version="1.0" encoding="utf-8"?>
 <configuration>
-    <startup> 
-        <supportedRuntime version="v4.0" sku=".NETFramework,Version=v4.6.1"/>
-    </startup>
-</configuration>
+<startup><supportedRuntime version="v4.0" sku=".NETFramework,Version=v4.6.2"/></startup></configuration>
```

Comment:
nit: would keep more readable formatting

Reply:
This was generated by the tool. I'll try to format these.